lib/begin.go

```diff
@@ -179,6 +190,9 @@ func (a *ACBuild) beginFromImage(start string, insecure bool) error {
 
 	err = reg.Fetch(app.Name, labels, 0, false)
 	if err != nil {
+		if _, ok := err.(*url.Error); ok {
+			return fmt.Errorf("unknown host when fetching image, local file paths must start with '/' or '.'")
```

Comment:
I followed the rabbit hole down as far as I could, and I think the check I have for it is as best I can get. Notably the exact same error gets returned for both when there is no internet connection and when there is a connection and the DNS lookup failed. I've modified the returned error to reflect this.